.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2.0
+jobs:
+  unit-tests:
+    docker:
+      - image: circleci/openjdk:15-jdk-buster
+
+    steps:
+      - checkout
+      - run:
+          name: Run unit tests
+          command: |
+            ./gradlew test
+      - store_test_results:
+          path: build/test-results
```

Comment:
What is this used for?

Reply:
That's so circleci picks up the test XML output to parse for the view [here](https://app.circleci.com/pipelines/github/openstax/xhtml-validator/7/workflows/9aa82a1a-472a-47c0-8d47-86c4aa9e6a78/jobs/7/tests).